<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timewarp</class>
 <widget class="QWidget" name="timewarp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timewarp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="add_pushButton">
     <property name="text">
      <string>Add Selected Objects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="remove_pushButton">
     <property name="text">
      <string>Removed Selected Objects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="get_all_pushButton">
     <property name="text">
      <string>Select All Timewarped Objects</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bake_pushButton">
     <property name="text">
      <string>Bake Timewarp</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="delete_pushButton">
     <property name="text">
      <string>Remove Timewarp</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
